docs/admin/kubernetes_deployment.md

```diff
@@ -0,0 +1,37 @@
+# Kubernetes Deployment Installation Guide
+Kserve supports `RawDeployment`, `RawDeployment` is based on kubernetes `Deployment`, `Service` and `Ingress`.
+
+Kubernetes 1.17 is the minimally recommended version.
+## 1. Install Istio
+The minimally required Istio version is 1.9.5 and you can refer to the [Istio install guide](https://knative.dev/docs/admin/install/installing-istio).
```

Comment:
Probably should point to installation guide from istio as knative is not relevant here.